lib/api/v1/report-controller.js

```diff
@@ -30,6 +31,30 @@ module.exports = function(app, user) {
     }
   });
 
+  // Load batch
+  app.get('/api/v1/report/batch', user.can('view data'), function(req, res) {        
+    batch.load(req.session.user._id, function(err, reports) {
+      if (err) res.send(err.status, err.message);
+      else res.send(200, {results: reports, total: reports.length});
+    });
+  });
+
+  // Checkout new batch
+  app.patch('/api/v1/report/batch', user.can('edit reports'), function(req, res) {    
+    batch.checkout(req.session.user._id, function(err, reports) {
+      if (err) res.send(err.status, err.message);
+      else res.send(200, {results: reports, total: reports.length});
+    });
+  });
+
+  // Cancel batch
+  app.put('/api/v1/report/batch', user.can('edit reports'), function(req, res) {
```

Comment:
Usually PUT is used to replace a full record. I see you're using it here to distinguish between checkout and cancel but that seems a bit non-standard to me. What about DELETE /report/batch?